docs/resources/samples/range-samples.md

```diff
@@ -0,0 +1,320 @@
+          ["PE", "Prince Edward Island", "CAN"],
+          ["QC", "Quebec", "CAN"],
+          ["SK", "Saskatchewan", "CAN"],
+          ["YT", "Yukon Territory", "CAN"]];
+}
+```
+
+## Change an adjacent cell
+
+This script gets adjacent cells using relative references. Note that if the active cell is on the top row, part of the script fails, because it references the cell above the currently selected one.
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook) {
+  // Get the currently active cell in the workbook.
+  let activeCell = workbook.getActiveCell();
+  console.log(`The active cell's address is: ${activeCell.getAddress()}`);
+
+  // Get the cell to the right of the active cell and set its value and color.
+  let rightCell = activeCell.getOffsetRange(0,1);
+  rightCell.setValue("Right cell");
+  console.log(`The right cell's address is: ${rightCell.getAddress()}`);
+  rightCell.getFormat().getFont().setColor("Magenta");
+  rightCell.getFormat().getFill().setColor("Cyan");
+
+  // Get the cell to the above of the active cell and set its value and color.
+  // Note that this operation will fail if the active cell is in the top row.
+  let aboveCell = activeCell.getOffsetRange(-1, 0);
+  aboveCell.setValue("Above cell");
+  console.log(`The above cell's address is: ${aboveCell.getAddress()}`);
+  aboveCell.getFormat().getFont().setColor("White");
+  aboveCell.getFormat().getFill().setColor("Black");
+}
+```
+
+## Change all adjacent cells
+
+This script copies the formatting in the active cell to the neighboring cells. Note that this script only works when the active cell isn't on an edge of the worksheet.
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook) {
+  // Get the active cell.
+  let activeCell = workbook.getActiveCell();
+
+  // Get the cell that's one row above and one column to the left of the active cell.
+  let cornerCell = activeCell.getOffsetRange(-1,-1);
+
+  // Get a range that includes all the cells surrounding the active cell.
+  let surroundingRange = cornerCell.getResizedRange(2, 2)
+
+  // Copy the formatting from the active cell to the new range.
+  surroundingRange.copyFrom(
+    activeCell, /* The source range. */
+    ExcelScript.RangeCopyType.formats /* What to copy. */
+    );
+}
+```
+
+## Change each individual cell in a range
+
+This script loops over the currently select range. It clears the current formatting and sets the fill color in each cell to a random color.
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook) {
+  // Get the currently selected range.
+  let range = workbook.getSelectedRange();
+
+  // Get the size boundaries of the range.
+  let rows = range.getRowCount();
+  let cols = range.getColumnCount();
+
+  // Clear any existing formatting
+  range.clear(ExcelScript.ClearApplyTo.formats);
+
+  // Iterate over the range.
+  for (let row = 0; row < rows; row++) {
+    for (let col = 0; col < cols; col++) {
+      // Generate a random color hex-code.
```

Comment:
Update to hexadecimal per [writing style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33958)

```suggestion
      // Generate a random hexadecimal color code.
```